lib/integrations/github.ts

```diff
@@ -796,6 +802,13 @@ module.exports = class GitHubIntegration implements Integration {
 				this.options,
 			);
 
+			if (!('type' in contractFile.data && 'content' in contractFile.data)) {
```

Comment:
@Hades32 What is `content` expected to be and is it really a required field? Did a quick search and found an example in our docs that doesn't appear to have a `content` defined: https://www.balena.io/docs/learn/deploy/deploy-with-balena-button/#balenayml-configuration-file

Reply:
@joshbwlng `contractFile` is from the GH API and can either be a file or a directory or other weird things. Therefore the typings are really crazy for that function. Checking for those fields was the best I could come up with to ensure it really is a file. `content` then contains the file contents. See
https://github.com/product-os/jellyfish-plugin-github/pull/577/files#diff-8ba77720a13b7e0118044336dd8c25fe8b5f4cf9a2349fa13e60acb97964412eR821-R823